src/app/store.ts

```diff
@@ -1,10 +1,20 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
 // eslint-disable-next-line import/no-cycle
 import counterReducer from '../features/counter/counterSlice';
+import usersReducer from '../features/users';
```

Comment:
Ensure that the import path '../features/users' correctly points to the file where the `usersReducer` is defined. If the path is incorrect, it will cause runtime errors.